.eleventy.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function (eleventyConfig) {
+  eleventyConfig.setUseGitIgnore(false);
+  eleventyConfig.addPassthroughCopy("js/");
```

Comment:
Why would this be necessary? Surely JS files are copied anyway aren't they?

Reply:
11ty keeps giving me a “document not defined” error for the JS libraries, if I leave it inside of the src folder. Which obviously it is not defined, because it is not being rendered in the browser. So the way I solved it is to avoid it to compile by copying directly to the final path.

This also addresses #pullrequestreview-456432683

Reply:
Strangely, moving the `defer` attribute in the `script` tag solved everything.

Reply:
So I have a tried every single script in `package.json` and it's the `dotrun serve` that is failing, which is basically eleventy rendering the files and templates. There is a similar (opened issue in their repo)[https://github.com/11ty/eleventy/issues/1088] and the workaround suggested is the the one I added.